cogs/weightcog.py

```diff
@@ -557,6 +560,45 @@ async def export(self, ctx: commands.Context, user: discord.Member = None) -> No
                 file=discord.File(csv_file, filename=f"{user.display_name}_weight_data.csv"),
             )
 
+    @commands.command()
+    async def del_all_log(self, ctx: commands.Context, user: discord.Member = None) -> None:
+        """
+        Deletes all log data (CSV file) associated with a specified Discord user or the command invoker.
```

Comment:
Also specify that only the data from the weight cog gets deleted. Other data from this user should not be touched.

Reply:
Added the required description.